### ISSUE TITLE
User side of the app blows up when admin opens a stretch

### ISSUE THREAD

**Issue by dm1031** [snippet-good/snippet-good#78]
When a user and admin of the same cohort are both logged in at the same time, the user side of the app blows up when the admin opens the stretch. It requires a hard reload in order to see the open stretch.

**Comment by ydhan18** [comment]
I agree. Also, when the open stretch runs out of time, the admin receives a flash message for each student, which clutters the admin's interface.

![Screen Shot 2019-06-23 at 1 08 11 PM (2)](https://user-images.githubusercontent.com/39353049/59979615-280d6b80-95b8-11e9-8b49-563fce09d9f9.png)
